app/src/renderer/components/staking/PageStaking.vue

```diff
@@ -76,19 +76,13 @@ export default {
         .slice(0, 100)
         .reduce((sum, v) => sum + v.voting_power, 0)
     },
-    enrichedAndFilteredDelegates() {
-      return this.filteredDelegates.map(d =>
-        Object.assign({}, d, {
-          your_votes: this.num.prettyInt(this.committedDelegations[d.id])
-        })
-      )
-    },
     filteredDelegates() {
       let query = this.filters.delegates.search.query || ""
 
       forEach(this.delegates.delegates, v => {
         v.small_moniker = v.moniker.toLowerCase()
         v.percent_of_vote = num.percent(v.voting_power / this.vpTotal)
+        v.your_vote = this.num.prettyInt(this.committedDelegations[v.id])
```

Comment:
you are mutating the delegates object at this place. We don't know if this could cause side effects. How about a computed property `enrichedDelegates` similar to `enrichedAndFilteredDelegates` you then can use in `filteredDelegates`?